crates/kitsune-core/src/mapping/mastodon/sealed.rs

```diff
@@ -589,3 +613,37 @@ impl IntoMastodon for DbNotification {
         })
     }
 }
+
+#[async_trait]
+impl IntoMastodon for (DbCustomEmoji, DbMediaAttachment, Option<Timestamp>) {
+    type Output = CustomEmoji;
+
+    fn id(&self) -> Option<Uuid> {
+        Some(self.0.id)
+    }
+
+    async fn into_mastodon(self, state: MapperState<'_>) -> Result<Self::Output> {
```

Comment:
Consider destructuring `self` inside the block for more readable code.

I.e `let (emoji, attachment, timestamp) = self;`